### ISSUE TITLE
iOS 9 compatability/readiness?

### ISSUE THREAD

**Issue by khappucino-nordstrom** [EddyBorja/MLPAutoCompleteTextField#83]
We're auditing our code for iOS 9. We're wondering if there's any known breaking changes in the new release. Thanks!

**Comment by andela-Kshittu** [comment]
Yes there is, the autocomplete tableview is coming up with empty suggestions.
